tests/phpunit/tests/block-supports/layout.php

```diff
@@ -252,4 +252,92 @@ public function data_layout_support_flag_renders_classnames_on_wrapper() {
 			),
 		);
 	}
+
+	/**
+	 * Check that wp_restore_group_inner_container() restores the legacy inner container on the Group block.
+	 *
+	 * @dataProvider data_restore_group_inner_container
+	 *
+	 * @covers ::wp_restore_group_inner_container
```

Comment:
```suggestion
	 * @ticket 60130 
	 *
	 * @covers ::wp_restore_group_inner_container
	 *
	 * @dataProvider data_restore_group_inner_container
```

Missing ticket number.